includes/Modules/Sign_In_With_Google.php

```diff
@@ -340,6 +341,42 @@ public function get_debug_fields() {
 		return $debug_fields;
 	}
 
+	/**
+	 * Gets map of datapoint to definition data for each endpoint.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array Map of datapoints to their definitions.
+	 */
+	protected function get_datapoint_definitions() {
+		return array(
+			'GET:anyone-can-register' => array( 'service' => 'sign-in-with-google' ),
```

Comment:
I see this is part of the IB but it seems like overkill for surfacing a site-wide setting. Why aren't we adding this to base data and surfacing it through `core/site`?

Reply:
Yeah that's a fair point, I'll make that change (it might require changing some fixtures/base tests but that's cool).